docs/modules/pipelines/pages/configuring-jobs.adoc

```diff
@@ -167,24 +167,3 @@ cluster change depending on these two settings.
 |suspend job
 |===
 
-== Storing Metrics
```

Comment:
It is a matter of taste where to put this section (in particular the table with `metricsEnabled` and `storeMetricsAfterJobCompletion`). In `configuring-jobs` it makes documentation of `JobConfig` complete. In `job-monitoring` it makes the job monitoring description complete and gives a more natural flow of reading (how to configure and use in one place). I do not have a strong opinion where it fits better or if it should be duplicated or linked.